keras_rcnn/models.py

```diff
@@ -1,33 +1,40 @@
 # -*- coding: utf-8 -*-
 
 import keras
+import keras_resnet
 import keras_rcnn.layers
+import keras_rcnn.heads
 
 
 class RCNN(keras.models.Model):
```

Comment:
@0x00b1 Actually, I think we should go for two arguments: features and heads (maybe rename these two to keep consistency).